Common/models/TopicResponse.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TopicResponse.swift
+//  Canillitapp
+//
+//  Created by Marcos Griselli on 21/07/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import Foundation
+
+struct TopicResponse: Decodable {
+    let keywords: [String]
+    let news: [String: [News]]
+    
+    func topics(date: Date) -> [Topic] {
+        return news.map { news -> Topic in
+            let topic = Topic()
+            topic.name = news.key
+            topic.date = date
+            topic.news = news.value
+            return topic
+        }
+    }
+}
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/ea7c45323b85f60b89a8c5fccaad3cedd57356b6/Common/models/TopicResponse.swift#L25"></span>Files should have a single trailing newline.</td>
    </tr>
  </tbody>
</table>